hotness/consumers.py

```diff
@@ -346,9 +346,8 @@ def _handle_anitya_update(self, upstream, package, msg):
                 self.bugzilla.follow_up(str(e), bz)
             except Exception as e:
                 _log.exception(e)
-                note = ("An unexpected error occured creating the scratch build: "
-                        "please report this issue to the-new-hotness issue tracker "
-                        "at https://github.com/fedora-infra/the-new-hotness/issues")
+                note = ("An unexpected error occurred creating the scratch build "
```

Comment:
Hmm. I _think_ both are valid English (or at least, they pass my internal "that sounds right" checks). I'm guessing the second version sounds better to non-native speakers so I'll change it.